src/data/markdown/docs/01 guides/02 Using k6/02 Metrics.md

```diff
@@ -61,17 +62,17 @@ In the above snippet, `res` is an [HTTP Response](/javascript-api/k6-http/respon
 
 | Property                      | Description                                                           |
 | ----------------------------- | --------------------------------------------------------------------- |
-| `res.body`                    | `string` containing the HTTP response body                            |
-| `res.headers`                 | `object` containing header-name/header-value pairs                    |
-| `res.status`                  | `integer` containing HTTP response code received from server          |
-| `res.timings`                 | `object` containing HTTP timing information for the request in **ms** |
-| `res.timings.blocked`         | = `http_req_blocked`                                                  |
-| `res.timings.connecting`      | = `http_req_connecting`                                               |
-| `res.timings.tls_handshaking` | = `http_req_tls_handshaking`                                          |
-| `res.timings.sending`         | = `http_req_sending`                                                  |
-| `res.timings.waiting`         | = `http_req_waiting`                                                  |
-| `res.timings.receiving`       | = `http_req_receiving`                                                |
-| `res.timings.duration`        | = `http_req_duration`                                                 |
+| res.body                    | `string` containing the HTTP response body                            |
+| res.headers                 | `object` containing header-name/header-value pairs                    |
+| res.status                  | `integer` containing HTTP response code received from server          |
+| res.timings                 | `object` containing HTTP timing information for the request in **ms** |
+| res.timings.blocked         | = `http_req_blocked`                                                  |
+| res.timings.connecting      | = `http_req_connecting`                                               |
+| res.timings.tls_handshaking | = `http_req_tls_handshaking`                                          |
+| res.timings.sending         | = `http_req_sending`                                                  |
+| res.timings.waiting         | = `http_req_waiting`                                                  |
+| res.timings.receiving       | = `http_req_receiving`                                                |
+| res.timings.duration        | = `http_req_duration`                                                 |
```

Comment:
Why remove the backticks from the property column?

Reply:
![Screenshot 2021-03-16 at 09 53 22](https://user-images.githubusercontent.com/825430/111281669-7ccf0080-863d-11eb-9252-60f69a199388.png)

I removed backticks to allow the `<sup> xxx </sup>` style.

![Screenshot 2021-03-16 at 09 55 10](https://user-images.githubusercontent.com/825430/111281912-ba338e00-863d-11eb-9d30-3a8e7fafdceb.png)